client/Packages/com.beamable.server/Editor/Usam/Editor/CsharpProjectUtil.cs

```diff
@@ -146,7 +146,6 @@ static IEnumerable<string> GetValidDllReferences(Assembly assembly)
 
 		static IEnumerable<Assembly> GetValidAssemblyReferences(Assembly assembly)
 		{
-			var invalidPrefixes = new string[] { "Unity.", "UnityEditor.", "UnityEngine." };

```

Comment:
did you mean to delete this? Won't this start generating invalid csproj files?

Reply:
Oh, it was an unused variable, the `IsValidReference ` already does that check ando also has these prefixes values.